src/web/components/ApiKeyManagement/KeyEditDialog.stories.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+/* eslint-disable camelcase */
+import type { Meta, StoryObj } from '@storybook/react';
+
+import KeyEditDialog from './KeyEditDialog';
+
+const meta: Meta<typeof KeyEditDialog> = {
+  component: KeyEditDialog,
+  title: 'Api Management/Edit Key Dialog',
+};
+export default meta;
+
+type Story = StoryObj<typeof KeyEditDialog>;
+
+export const MultipleRoles: Story = {
+  args: {
+    apiKey: {
+      contact: 'ApiKey',
+      name: 'ApiKey',
+      created: 1702830516,
+      key_id: 'F4lfa.fdas',
+      site_id: 1,
+      disabled: false,
+      roles: [
+        { id: 1, roleName: 'MAPPER', externalName: 'Mapper' },
+        { id: 2, roleName: 'GENERATOR', externalName: 'Generator' },
+      ],
+      service_id: 0,
+    },
+    availableRoles: [
+      { id: 1, roleName: 'MAPPER', externalName: 'Mapper' },
+      { id: 2, roleName: 'GENERATOR', externalName: 'Generator' },
+      { id: 3, roleName: 'ID_READER', externalName: 'Bidder' },
+      { id: 4, roleName: 'SHARER', externalName: 'Sharer' },
+    ],
+    onEdit: (form) => {
+      console.log(form);
+      return Promise.resolve();
+    },
+    triggerButton: <button type='button'>Open</button>,
+  },
+};
+
+export const KeyWithNotAllowedRoles: Story = {
```

Comment:
The name of this story is confusing me - what is it actually testing? I think it is testing the fact that it is showing Mapper, Generator and Bidder, despite Bidder not being one of the `apiKey.roles`? Maybe we could think of a better name for this story?

Reply:
Have renamed it to this: `KeyWithRolesParticipantIsntAllowed`. The tests aim is to show the behaviour when a key contains roles which the participant isn't actually allowed.